### DISCUSSION TITLE
[Bug?] Checkout.min.js not loaded on fresh install of wordpess docker and woocommerce 9.0.2

### DISCUSSION THREAD

**Discussion by benyaminl** [woocommerce/woocommerce#48840] [Checkout Flow]
Hello, I got a problem on checkout page, that broke one plugins that I would be use, WooOngkir. 

I seen that between working WooCommerce Instance that use WooOngkir, there are distinct js that isn't loaded, it's checkout.min.js

The problem Is I check the WooCommerce code, it should already enqued using `WC_Frontend_Scripts->register_scripts` at `wc-checkout` hooks... 

![image](https://github.com/woocommerce/woocommerce/assets/2817332/f8b0b007-3b42-465a-9482-adcfb0797ff2)

I'm pretty confused at this point because I tried multiple clean installation of the wooCommerce, nothing changed

I use the WP 6.5.5 from `docker.io/library/wordpress:latest`, and WooCommerce 9.0.2 from the WP Market (plugin search then install). Theme used/tested is 2024 and 2023 from Wordpress

EDIT : I Found that in 9.0.2 it start removing 

```
	/**
	 * Dequeues the scripts added by WC Core to the Checkout page.
	 *
	 * @return void
	 */
	public function dequeue_woocommerce_core_scripts() {
		// wp_dequeue_script( 'wc-checkout' );
		// wp_dequeue_script( 'wc-password-strength-meter' );
		// wp_dequeue_script( 'selectWoo' );
		// wp_dequeue_style( 'select2' );
	}
```

Is that means we already making a lot of old plugins? 

And what are the alternative to fix it? 

Is there anyone can enlighten me? 

Thank you

**Comment by nielslange** [comment]
Hello @benyaminl 👋

I checked the [Woongkir plugin page](https://wordpress.org/plugins/woongkir/) and found this notice:

> This plugin is not compatible with WooCommerce blocks. You MUST use [WooCommerce shortcodes](https://woo.com/document/woocommerce-shortcodes) to build your cart and checkout page.

If you’re unable to use the Woongkir plugin with WooCommerce blocks, I recommend reaching out to the plugin developer directly through [their support page](https://wordpress.org/support/plugin/woongkir/).

Since there are no actionable items for us, I'll close this discussion. However, feel free to start a new one if you have more to discuss. For reporting bugs or suggesting features, please use the following link to create an issue:

https://github.com/woocommerce/woocommerce/issues/new/choose

Discussions here should focus on gathering community feedback about potential changes.

**Reply by benyaminl** [reply]
Yeah, the shortcodes works. I think the support for guttenberg on the way. Thank you for giving me feedback.
